packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -110,6 +110,18 @@ $theme-identifier-max-width: "desktop" !default;
 $theme-identifier-primary-link-color: default !default;
 $theme-identifier-secondary-link-color: "base-light" !default;
 
+// In-Page Navigation
+$theme-in-page-nav-background-color: "default" !default;
+$theme-in-page-nav-background-padding: 2 !default;
+$theme-in-page-nav-background-radius: "lg" !default;
+$theme-in-page-nav-bar-color: "default" !default;
```

Comment:
Changing this value doesn't do anything &mdash;  I don't see that this variable is being used anywhere.

Reply:
ha whoops, lemme check!

Reply:
OK, fixed.